libs/ui/paginator/src/lib/paginator/paginator.component.ts

```diff
@@ -315,6 +317,18 @@ export class TsPaginatorComponent implements OnChanges, AfterViewInit {
   @Output()
   public readonly recordsPerPageChange = new EventEmitter<number>();
 
+  /**
+   * Emit an event when previous page button clicked
+   */
+  @Output()
+  public readonly prevPage = new EventEmitter();
```

Comment:
So far we have avoided most abbreviations in our code and API. My preference would be to keep that going and use something like `previousPage`

Reply:
Bad influence from other app when they had `prev()` all over lol

Reply:
😂